webpack.config.js

```diff
@@ -294,6 +300,7 @@ const webpackConfig = ( env, argv ) => {
 		// Build basic modules that don't require advanced optimizations, splitting chunks, and so on...
 		{
 			entry: {
+				'googlesitekit-i18n': './assets/js/googlesitekit-i18n.js',
```

Comment:
I have added it to the basic modules list to eliminate chunking. Otherwise, the original `@wordpress/i18n` package would appear in the shared `vendors` script.

Reply:
Makes sense, thanks for explaining.